### PR TITLE
Markdown conversion for `pt-BR` - Cleanup - HTTP section

### PR DESCRIPTION
**Pull request by SphinxKnight**
No conversion report as there is no error left.

### CODE REVIEW

**Review by josielrocha — APPROVED** [review]
LGTM ;)

File: files/pt-br/web/http/basics_of_http/mime_types/index.html [hunk 1]
```diff
@@ -305,11 +305,5 @@ <h2 id="See_also">See also</h2>
 
 <ul>
  <li><a href="/en-US/docs/Web/Security/Securing_your_site/Configuring_server_MIME_types">Properly configuring server MIME types</a></li>
- <li>
-  <p><a href="/en-US/docs/Web/HTML/Supported_media_formats">Media formats supported by the HTML audio and video elements</a></p>
- </li>
+ <li><a href="/en-US/docs/Web/HTML/Supported_media_formats">Media formats supported by the HTML audio and video elements</a></li>
```
Comment (nathipg) [inline files/pt-br/web/http/basics_of_http/mime_types/index.html:308]:
```suggestion
 <li><a href="/pt-BR/docs/Web/HTML/Supported_media_formats">Media formats supported by the HTML audio and video elements</a></li>
```

**Comment by nathipg** [comment]
@SphinxKnight Obrigada pela contribuição!

Percebi que todos os arquivos ainda estão com a extensão .html, acho que faltou mudar pra .md (Considerando o título do PR), poderia dar uma olhada nisso, por favor?

**Comment by nathipg** [comment]
@SphinxKnight Thank you for helping!

I noticed that all the files are still with the .html extension, I think it is necessary to change it to .md (Considering the PR title), could you take a look at this, please?

**Comment by SphinxKnight** [comment]
@nathipg hello, the goal of those PRs is actually to prepare the content for Markdown conversion, adapting any part which may result in conversion errors/the content being kept in HTML.
I do intend to propose conversion PRs once those "cleanup PRs" are merged :)

Please let me know if this is unclear.
See also:
- https://github.com/mdn/markdown/blob/main/conversion-guide.md#optional-submit-a-pr-with-the-html-cleanup
- https://github.com/mdn/translated-content/issues/7950

**Comment by SphinxKnight** [comment]
Merging this per @josielrocha approval.

**Pull request merged**